tests/rewards/test_reward_fn.py

```diff
@@ -0,0 +1,46 @@
+"""Tests `imitation.rewards.reward_function` and `imitation.rewards.serialize`."""
+
+import numpy as np
+import pytest
+
+from imitation.rewards import reward_function, serialize
+
+
+def _funky_reward_fn(obs, act, next_obs, done):
+    """Returns consecutive reward from 1 to batch size `len(obs)`."""
+    # give each environment number from 1 to num_envs
+    return (np.arange(len(obs))).astype("float32")
+
+
+obs = np.random.randint(0, 10, (64, 100))
+acts = next_obs = done = obs
```

Comment:
`done` is meant to be of boolean type. Of course any input to non-obs parameters works with `_funky_reward_fn`, but this is a bit confusing as written. Consider using `np.zeros(64, dtype=np.bool)` or just setting all of the non-obs to `None` (this is also illegal, but is at least more up-front about it).